### ISSUE TITLE
Unable to Tab Through Top Nav with OS X VoiceOver?

### ISSUE THREAD

**Issue by jpdevries** [modxcms/a11y#65]
The Added ARIA roles for trees commit looks awesome https://github.com/modxcms/a11y/commit/fb8b35a333acb040bedd789479bec974fca18fec

When testing with OS X VoiceOver I was the screen reader appears to get "stuck" at the MODX logo ([screencast](http://j4p.us/221K1S2A0S1c/a11y_busy.mov)).

**Comment by jpdevries** [comment]
Discovered in Safari you need to enable "tab through" support.

![](http://j4p.us/image/0U2q1d1R0Y1F/Screen%20Shot%202015-11-29%20at%203.03.48%20PM.png)

After doing so could tab trough Resource, Elements, Files tree.
